### DISCUSSION TITLE
More clear admin listings in general, categories specially

### DISCUSSION THREAD

**Discussion by madsoliver** [PrestaShop/PrestaShop#41078] [Ideas]
#### Is your feature request related to a problem?
- Merchants do not know how to sort categories.
- Merchants do not know how to edit a category with children.

#### Describe the solution you'd like
- I suggest showing a more standardized sorting symbol like: https://commons.wikimedia.org/wiki/File:Move_icon.svg
- I suggest showing the icons all the time, and not only on hover
- I suggest allways showing edit-button (not hidden in more/context/threedot-menu)
- I suggest showing "Show"-button as greyed out / disabled when there are no children
- I suggest also showing delete-button. 
- I suggest showing more/context/threedot only if a module or other has added more options here.

![Screenshot 2021-10-08 at 09 34 04](https://user-images.githubusercontent.com/30602638/136518437-c46047c5-5c22-4aac-9803-d6b800aab6af.png)

#### Alternatives you've considered
- I suggest doing similar things for all admin listings.

#### Additional context
PS 1.7.8.0

**Comment by khouloudbelguith** [comment]
Hi @onklmaps,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thanks!

**Comment by ElodieOS** [comment]
Hi everyone ! 
Thank you for yours suggestions @madsoliver :)

**I’m totally agree with the issue of the edit-button**. We can have it all the time because it is the main action we need to do quickly. 

However, for the other propositions, I have some coments: 
- I think the sorting icon is quite recognizable because it’s more and more used in modern software. Unfortunately, the table contains a lot of information, so to keep a quick use, you can't display everything. It's better to let only the main and frequently used actions appear, like the **edit-button which is a very good example that you have highlighted**. In this logic, it’s better to keep the sorting-button in hover and the delete-button in the dropdown menu
- For disable state of the show-button, it’s a good idea too, but unfortunately after doing the test, we don’t see enough difference between the two states. It could confuse users. By removing it when it’s not necessary, the information is clear and the table is less overloaded by showing only the necessary and possible actions

Here, is a mock-up of my proposition
![image](https://user-images.githubusercontent.com/113457393/193250267-32a6feca-0949-4e89-8f9b-bde52435ca94.png)
